Source/QED/BreitWheelerEngineWrapper.H

```diff
@@ -2,53 +2,279 @@
 #define WARPX_breit_wheeler_engine_wrapper_h_
 
 #include "QedWrapperCommons.H"
+#include "BreitWheelerEngineInnards.H"
 
-//BW ENGINE from PICSAR
+#include <AMReX_Array.H>
+#include <AMReX_Vector.H>
+#include <AMReX_Gpu.H>
+
+//#define PXRMP_CORE_ONLY allows importing only the 'core functions' of the
+//Breit Wheeler engine of the QED PICSAR library.
 #define PXRMP_CORE_ONLY
-#include "breit_wheeler_engine.hpp"
+#include <breit_wheeler_engine.hpp>
+
+//Lookup table building function is in a dedicated (optional) class to
+//avoid including heavy dependencies if they are not needed.
+#ifdef WARPX_QED_TABLE_GEN
+    #include "BreitWheelerEngineTableBuilder.H"
+#endif
+
+#include <string>
 
-using WarpXBreitWheelerWrapper =
-    picsar::multi_physics::breit_wheeler_engine<amrex::Real, DummyStruct>;
+//Some handy aliases
+using PicsarBreitWheelerEngine = picsar::multi_physics::
+    breit_wheeler_engine<amrex::Real, QedUtils::DummyStruct>;
```

Comment:
I've added some comments to explain why I am using an empty data structure there.
I think that having `DummyStruct` in `QedUtils` is the best solution, since it is used twice (in the two engine headers). An alternative could be to define something like `BWDummyStruct` and `QSDummyStruct` in the two headers.